pkg/proto/datadog/process/workloadmeta_process.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package datadog.process;
+
+option go_package = "pkg/proto/pbgo"; // golang
+
+message WorkloadMetaProcessEvents {
+  int32 eventID = 1;
+  repeated ProcessEventSet setEvents = 2;
+  repeated ProcessEventUnset unsetEvents = 3;
+}
+
+message ProcessEventSet {
+  int32 pid = 1;
+  int32 nspid = 2;
+  string containerId = 3;
+  int64 creationTime = 4;
+  Language language = 5;
+}
+
+message ProcessEventUnset {
+  int32 pid = 1;
+}
+
+message Language {
+  string name = 1;
+}
+
+service ProcessEntityStream {
+  rpc StreamEntities (ProcessStreamEntitiesReq) returns (stream WorkloadMetaProcessEvents);
+}
```

Comment:
Looking at how the existing WorkloadMeta stream service is defined ([link](https://github.com/DataDog/datadog-agent/blob/18199646624418e380aef109baf37832098b8106/pkg/proto/datadog/api/v1/api.proto#L127-L133)) and the gRPC standards ([doc](https://grpc.io/docs/what-is-grpc/core-concepts/#service-definition)), let's use `...Request` and `...Response` in the method.

```suggestion
service ProcessEntityStream {
  rpc StreamEntities (ProcessStreamRequest) returns (stream ProcessStreamResponse);
}
```